nmdc_schema/migrators/adapters/dictionary_adapter.py

```diff
@@ -218,7 +242,10 @@ def delete_documents_having_value_in_field(self, collection_name: str, field_nam
         # Filter out the documents having the specified value in the specified field.
         # Reference: https://docs.python.org/3/library/functions.html#filter
         documents_initial = self._db[collection_name]
-        document_generator = filter(lambda d: field_name not in d or d.get(field_name) != value, documents_initial)
+        document_generator = filter(
+            lambda d: field_name not in d or d.get(field_name) != value,
+            documents_initial,
+        )
```

Comment:
Note: This differs in format only — the code is the same as before, just reformatted.